test/Azure.Functions.Cli.Tests/Resources/DurableTestFolder/Counter.cs

```diff
@@ -71,5 +73,25 @@ public static async Task Increment([OrchestrationTrigger] DurableOrchestrationCo
 
             ctx.ContinueAsNew(counterState);
         }
+
+        [FunctionName("JsonInput")]
+        public static async Task JsonInput([OrchestrationTrigger] DurableOrchestrationContext ctx)
+        {
+            string input = ctx.GetInput<string>();
+
+            if (!string.IsNullOrEmpty(input))
+            {
+                JObject testParsingInput = JObject.Parse(input);
```

Comment:
As written, it seems the CLI command requires users to take strings as inputs and deserialize them into the object they want in their function code. But this is a hassle and the `DurableOrchestrationContext` is supposed to do that for them. I suggest we change the orchestration code to look more like your test code, e.g.:
```csharp
[FunctionName("HelloWorldOrchestrator")]
public static string HelloWorldOrchestrator([OrchestrationTrigger] DurableOrchestrationContext ctx)
{
    var input = ctx.GetInput<dynamic>();
    return $"{input.Hello}, {input.Name}";
}
```
This will require you to make the change to your input parsing method, which I mentioned earlier.